reference/features/search_parameters.md

```diff
@@ -299,25 +299,25 @@ If `attributesToCrop` is not configured, `cropLength` has no effect on the retur
 **Expected value**: an array of <clientGlossary word="attribute" label="attributes" /> or `["*"]`
 **Default value**: `null`
 
-Highlights matching query terms in the given attributes. When this parameter is set, the `_formatted` object is added to the response for each document, within which you can find the highlighted text.
+Highlights matching query terms in the specified attributes by enclosing them in `<em>` tags. `attributesToHighlight` only works on values of the following types: string, number, array, object.
```

Comment:
We should probably make this a list, right?
```suggestion
Highlights matching query terms in the specified attributes by enclosing them in `<em>` tags. 

`attributesToHighlight` only works on values of the following types:

- string
- number
- array
- object
```

Reply:
Isn't this giving a disproportionate amount of attention to a list of types not seen in other places in the docs?

Perhaps that's because I'm not on team "always use lists when you have more than two items"?

Reply:
Yeah, you could be right. I'm happy to leave this change out.